D3D12HelloWorld/Graphics/CommandQueue.cs

```diff
@@ -53,14 +53,11 @@ public class CommandQueue : IDisposable {
         }
 
         public void Dispose() {
+            mFenceEvent.Dispose();
             mFence.Dispose();
             mCommandQueue.Dispose();
         }
 
-        [Obsolete("Work to refactor off this in favour of ExecuteCommandLists(CompiledCommandList), which accepts the higher CompiledCommandList abstraction.")]
-        internal void ExecuteCommandList(ID3D12GraphicsCommandList commandList)
-            => mCommandQueue.ExecuteCommandList(commandList);
-
```

Comment:
Now refactored out, D3D12Mutiny uses the higher CommandList abstraction for better DX12GE architecture compatibility